src/decl.jl

```diff
@@ -0,0 +1,150 @@
+# !!! Note:
+#    Many thanks to `@matthieubulte` for this contribution to `SymPy`.
+
+# The map_subscripts function is stolen from Symbolics.jl
```

Comment:
What are the licenses for these codes?

Reply:
Good question, MIT (https://github.com/JuliaPy/SymPy.jl/blob/master/LICENSE.md)

Reply:
and https://github.com/JuliaSymbolics/Symbolics.jl/blob/master/LICENSE.md